assignment/assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_instructor_list_submissions.vm

```diff
@@ -630,6 +653,17 @@
 								#end
 								#end
 							</td>
+							#if($!isAdditionalNotesEnabled && !$assignment.isGroup())
+							<td headers="notes">
+								#set($notes = [])
+								#set($notes = $!candidateDetailProvider.getAdditionalNotes($siteUser, $site).orElse([]))
+								<ul>
+								#foreach($note in $notes)
+									#if($!note) <li>$!note</li> #end
```

Comment:
Do you mean higher in the logic as in Java instead of vm? That's pretty hard as the user-submitter objects don't have the notes field, and adding it to the API doesn't seem a great idea. An alternative would be a map of submitterId/notes, but we decided this other way (also seen somewhere else on assignments). I guess that's why you said 'Ideally' and maybe this is not a mandatory correction?